plugin-server/src/worker/ingestion/lazy-groups-container.ts

```diff
@@ -0,0 +1,32 @@
+import { DB, GroupId, GroupsData } from '../../utils/db/db'
+
+export class LazyGroupsContainer {
```

Comment:
Problem: This abstraction _only_ works if you also update the container if an update happens somewhere. This isn't done currently and would result in incorrect properties/created_at being stored on $groupidentify

Reply:
nice, will update